PNA.mdk

```diff
@@ -376,32 +376,25 @@ here as "main".
 
 ~ TableFigure {#results-of-one-pkt-thru-main; \
   caption: Result of packet processed one time by main.}
-|-------------|---------------|--------------------|
-|             | Processed     | Resulting          |
-| Description | next by       | packet(s)          |
-+:------------+:--------------+:-------------------+
-| packet from   | main, with    | Zero or more mirrored  |
-| network port  | direction     | packets, plus at most  |
-|---------------|               | |
-| packet from   | `NET_TO_HOST` | one of: a net-to-host  |
-| net-to-host   |               | recirculated packet,   |
-| recirculate   |               | or one to-host packet. |
-|---------------|               |                        |
-| packet from   |               |                        |
-| port loopback |               |                        |
 |---------------|---------------|------------------------|
-| packet from   | main, with    | Zero or more mirrored  |
-| message processing | direction | packets, plus at most |
-|---------------|               | |
-| packet from   | `HOST_TO_NET` | one of: a host-to-net  |
-| host-to-net   |               | recirculated packet,   |
-| recirculate   |               | or one to-net packet.  |
+|               | Processed     | Resulting              |
+| Description   | next by       | packet(s)              |
++:--------------+:--------------+:-----------------------+
+| packet from   | main          | Zero or more mirrored  |
+| network port  |               | packets                |
```

Comment:
In the "Resulting packet(s)" column, would it be more accurate to expand it to something like the following text?

Zero or more mirrored packets, plus at most one copy of the original packet (which may be marked for recirculation).

Reply:
Are the mirrored packets also allowed to be recirculated? I would think yes, as it may be the only way to support broadcast traffic to a number of VMs larger than the target's maximum mirror slots.

Is the original packet always emitted albeit possibly marked for drop? (The comment in the P4 file says it is dropped after the end of the control block.)

If both the above are true, I would suggest this text:

> Original packet which may be recirculated or marked for drop, and zero or more mirrored packets some of which may be recirculated.

I'm also adding a forward reference to the packet mirroring section (§6.8) in the corresponding paragraph just after this table.